### PR TITLE
Greetings

### LINKED ISSUE #1: Needs more greetings.
Existing greetings kind of lame. Improve.

### PR DESCRIPTION
**Pull request by glebec**
Closes #1.

### CODE REVIEW

**Comment by glebec** [comment]
Hey @star-lebec whaddya think?

File: browser/js/common/factories/RandomGreetings.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ app.factory('RandomGreetings', function () {
     };
 
     var greetings = [
+        'GHA 4EVA!',
```
Comment (glebec) [inline browser/js/common/factories/RandomGreetings.js]: This is really great don't you think?

Comment (star-lebec) [inline browser/js/common/factories/RandomGreetings.js]: Needs more dog-accepting quotes.

File: browser/js/common/factories/RandomGreetings.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ app.factory('RandomGreetings', function () {
     };
 
     var greetings = [
+        'woof woof!',
```
Comment (star-lebec) [inline browser/js/common/factories/RandomGreetings.js]: Perfect! Woof woof

**Pull request merged**